pkgs/development/libraries/librsvg/default.nix

```diff
@@ -95,11 +106,13 @@ stdenv.mkDerivation rec {
         -i gdk-pixbuf-loader/librsvg.thumbnailer.in
   '';
 
-  # Merge gdkpixbuf and librsvg loaders
   postInstall = ''
-    mv $GDK_PIXBUF/loaders.cache $GDK_PIXBUF/loaders.cache.tmp
-    cat ${gdk-pixbuf.out}/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache $GDK_PIXBUF/loaders.cache.tmp > $GDK_PIXBUF/loaders.cache
-    rm $GDK_PIXBUF/loaders.cache.tmp
+    # Not generated when cross-compiling.
```

Comment:
We rely on this ugly hack for `wrapGAppsHook`, without which SVG icons will not work in GTK (unless the cache is not generated globally by NixOS module, which is shady too IMO). But I guess it does not matter for using librsvg directly as a library/`rsvg-convert`.

Reply:
This is also a good smoke-check to see if the loader cache is actually generated. I think it's better to put this behind `lib.optionalString (stdenv.buildPlatform != stdenv.hostPlatform)` rather than checking if the file exists. Now it would happily work even when the loader cache doesn't exist in the 'normal' situation.